src/raftstore/store/worker/apply.rs

```diff
@@ -812,6 +830,7 @@ impl ApplyDelegate {
         if let Err(e) = write_peer_state(&self.engine, ctx.wb, &region, state) {
             panic!("{} failed to update region state: {:?}", self.tag, e);
         }
+        self.flush_wal = true;
```

Comment:
I think we can set flush WAL to true when we meet any admin command. no need to set true in these admin commands.